### ISSUE TITLE
Top Locations in Wien

### ISSUE THREAD

**Issue by brodecva** [odalic/sti#180]
https://grips.semantic-web.at/x/sASBB

**Comment by josef-janousek** [comment]
Version: develop 11.12.2016
File: http://data.wien.gv.at/csv/top-locations-wien.csv

- Note: The original file is not correctly formatted. Some rows (for example row 17) contain one more semicolon in the column Beschreibung, so that they have one more column and input cannot be correctly processed. The UI displayed the error statement, but user cannot recognize, that the error was caused by bad formatting of the input file:
![top-locations-wien_error](https://cloud.githubusercontent.com/assets/17772934/21082546/edfb249e-bfdd-11e6-9e00-bc9950ad8acf.png)

Then I removed this superfluous semicolon.
After a long time of run, another error was thrown:
![top-locations-wien-error2](https://cloud.githubusercontent.com/assets/17772934/21082573/5733d64a-bfde-11e6-8680-9c4d3bcf1225.png)
